js/packages/logger/src/logger.ts

```diff
@@ -36,6 +37,11 @@ export function makeLogger({
   console = true,
   extended = false,
 }: LoggerOptions & {extended?: boolean} = {}): Logger {
+  if (process.env.APPLITOOLS_SHOW_LOGS && process.env.DEBUG) {
+    handler = {type: 'debug', label}
+    level = LogLevel.all
+    format = (chunks: any[], {tags, level}) => defaultFormat(chunks, {level, prelude: true, tags, timestamp: false})
```

Comment:
Instead of doing this, there is an easy configure all of this in `makePrinter`, you can check out what argument we are passing there.

Reply:
@kyryloonufriiev  I'm not sure what you want me to do with `makePrinter` here... can you please explain?

Reply:
@zamboney Why do you need to override format if everything you can do is could be configured in the printer, just take a look at how it is organized there. You can simply override the arguments you want to be different when debug handler is applied. For example, you want the timestamp to be `false` (you set it in your formatter), so you can just do `timestamp = false`. Then you don't want a label so you can do `label = false`

Reply:
@kyryloonufriiev I don't understand the difference

Reply:
You have a way to configure what you need. I don't understand the reason to invent another one.

Reply:
Printer handler that has been created (a.k.a. has log method) so using the Printer can't be used at this stage....

Reply:
Wouldn't it achieve exactly what you need, but in the way it was intended?
```suggestion
    timestamp = false
    label = null
```

Reply:
used `makePrinter` please take a look 10x